core/src/main/java/org/mule/runtime/core/internal/processor/strategy/ProactorStreamProcessingStrategyFactory.java

```diff
@@ -43,46 +43,35 @@
  */
 public class ProactorStreamProcessingStrategyFactory extends ReactorStreamProcessingStrategyFactory {
 
-  private static final ReactorProcessingStrategyFactory NOT_CONCURRENT_TX_AWARE_PS_FACTORY =
-      new ReactorProcessingStrategyFactory();
-
   @Override
   public ProcessingStrategy create(MuleContext muleContext, String schedulersNamePrefix) {
-    if (getMaxConcurrency() == 1) {
-      return NOT_CONCURRENT_TX_AWARE_PS_FACTORY.create(muleContext, schedulersNamePrefix);
-    } else {
-      return new ProactorStreamProcessingStrategy(() -> muleContext.getSchedulerService()
-          .customScheduler(muleContext.getSchedulerBaseConfig()
-              .withName(schedulersNamePrefix + RING_BUFFER_SCHEDULER_NAME_SUFFIX)
-              .withMaxConcurrentTasks(getSubscriberCount() + 1)),
-                                                  getBufferSize(),
-                                                  getSubscriberCount(),
-                                                  getWaitStrategy(), () -> muleContext.getSchedulerService()
-                                                      .cpuLightScheduler(muleContext.getSchedulerBaseConfig()
-                                                          .withName(schedulersNamePrefix + "." + CPU_LITE.name())),
-                                                  () -> muleContext.getSchedulerService()
-                                                      .ioScheduler(muleContext.getSchedulerBaseConfig()
-                                                          .withName(schedulersNamePrefix + "." + BLOCKING.name())),
-                                                  () -> muleContext.getSchedulerService()
-                                                      .cpuIntensiveScheduler(muleContext.getSchedulerBaseConfig()
-                                                          .withName(schedulersNamePrefix + "." + CPU_INTENSIVE.name())),
-                                                  getMaxConcurrency());
-    }
+    return new ProactorStreamProcessingStrategy(() -> muleContext.getSchedulerService()
+        .customScheduler(muleContext.getSchedulerBaseConfig()
+            .withName(schedulersNamePrefix + RING_BUFFER_SCHEDULER_NAME_SUFFIX)
+            .withMaxConcurrentTasks(getSubscriberCount() + 1)),
```

Comment:
what was the reason for this `+1`? I know it was in the previous code, but can you add a comment?